package.json

```diff
@@ -17,16 +17,18 @@
     "deploy": "npm run build && npm run gh-pages -- -m 'Updates: Manual build'"
   },
   "dependencies": {
-    "@mantine/core": "^4.2.2",
-    "@mantine/hooks": "^4.2.2",
-    "gatsby": "^4.13.1",
-    "gatsby-plugin-google-analytics": "^4.14.0",
-    "gatsby-plugin-manifest": "^4.14.0",
+    "@lekoarts/gatsby-theme-minimal-blog": "^4.1.7",
+    "@mantine/core": "^4.2.10",
+    "@mantine/hooks": "^4.2.10",
+    "gatsby": "^4.16.0",
+    "gatsby-plugin-google-analytics": "^4.16.0",
+    "gatsby-plugin-manifest": "^4.16.0",
     "gatsby-plugin-mantine": "^4.0.0",
-    "react": "^18.1.0",
```

Comment:
do we really need to downgrade react? 
the original lekoarts engine also depends on 18 https://github.com/LekoArts/gatsby-starter-minimal-blog/blob/master/package.json

Reply:
You're right however to merge  our website together it needed a dependency called "gatsyby-plugin-mdx"
which requires react version 17 or 16.  If you try to upgrade react to version 18 you will get the error shown below. 
![PeerDependency-gatsby-plugin](https://user-images.githubusercontent.com/24616310/174858233-ab96aefd-8701-48ef-a2ce-711258542c68.png)